apps/webapp/app/v3/services/resumeTaskRunDependencies.server.ts

```diff
@@ -34,14 +34,16 @@ export class ResumeTaskRunDependenciesService extends BaseService {
       return;
     }
 
-    const { batchItem, dependency } = taskAttempt.taskRun;
+    const { batchItems, dependency } = taskAttempt.taskRun;
 
-    if (!batchItem && !dependency) {
+    if (!batchItems.length && !dependency) {
       return;
```

Comment:
Is this a bug with the system and should we log something?

Reply:
Nope, this just means theres no dependencies to resume